src/pages/gear.page.tsx

```diff
@@ -1,11 +1,85 @@
+import { useEffect, useState } from 'react'
+import { ParchmentInput } from '../components/ParchmentInput'
 import { PageHeader } from '../components/page-header'
 import { Parchment } from '../components/parchment'
 import { rollD6 } from '../functions/dice.functions'
+import { ParchmentButton } from '../components/ParchmentButton'
+import { ArrowPathIcon } from '@heroicons/react/24/outline'
+import { t } from 'i18next'
+import { useTranslation } from 'react-i18next'
 
 export const GearPage = () => {
+  const { t, i18n } = useTranslation(['gear'])
+  const [searchGoods, setSearchGoods] = useState('')
+  const [maxPrice, setMaxPrice] = useState<number>(Infinity)
+  const [itemAvailability, setItemAvailability] = useState<Record<
+    string,
+    Availability
+  > | null>(null)
+
+  // create a filterarray of goods that match the search string and/or max price
```

Comment:
I don't think that the comment adds anything to the code. And there is really no reason to explain "why" we filter the goods.
```suggestion
```